.github/workflows/quarto-check.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+name: Check Quarto Render
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Quarto
+        uses: quarto-dev/quarto-actions/setup@v2
+
+      - name: Dependencies
+        run: quarto install tinytex
```

Comment:
Maybe it doesn't matter but is it `quarto install tinytex` or `quarto install tool tinytex`? (See: https://github.com/quarto-dev/quarto-actions/tree/main/setup#:~:text=quarto%20install%20tool%20tinytex)

Reply:
ah looks like it's just a var within setup, ill change